app/models/Ticket.scala

```diff
@@ -0,0 +1,21 @@
+package models
+
+case class Ticket(rowNumber: Int, seatNumberInRow: Int, ticketType: TicketType) {
+  override def toString: String = s"Row number: $rowNumber <--> seat number: ${seatNumberInRow + 1} <--> ticket type: $ticketType <--> ticket's price: ${ticketType.price} zł"
+}
+
+abstract class TicketType {
```

Comment:
I'd use `sealed abstract class` here. Do you know why?

Reply:
Yes, to make sure that it would be impossible to create a class that extends TicketType in any other file. Using "sealed" just helps developers so that all subclasses are visible in the same file - better readability and less time needed to understand how it works.